src/fw-update/fw-update-device.cpp

```diff
@@ -210,8 +213,14 @@ namespace librealsense
         // WaitForDFU state sends several DFU_GETSTATUS requests, until we hit
         // either RS2_DFU_STATE_DFU_MANIFEST_WAIT_RESET or RS2_DFU_STATE_DFU_ERROR status.
         // This command also reset the device
-        if (!wait_for_state(messenger, RS2_DFU_STATE_DFU_MANIFEST_WAIT_RESET, 20000))
-            throw std::runtime_error("Firmware manifest failed");
+        
+        if (d500_device)
+            // TODO: DFU issue - for HKR, sleep 80 seconds till finish writing to qspi flash
+            // and wake up the device with the new burned FW
+            // HKR doesn't need the next check below (RS2_DFU_STATE_DFU_MANIFEST_WAIT_RESET)
+            std::this_thread::sleep_for(std::chrono::milliseconds(80000));
+        else if (!wait_for_state(messenger, RS2_DFU_STATE_DFU_MANIFEST_WAIT_RESET, 20000))
```

Comment:
Please verify the error code we get and send to Gang.
Not sure why we get an error on HKR and not on DS5

Reply:
it is RS2_USB_STATUS_OTHER (-13) as I said before.
Sent on teams.

Reply:
As I wrote on teams, it may be the effect of a slow wakeup of the HKR, it's a USB error.
Lets try to allow retries for that, meaning skip the return false and let it sleep and iterate for some time until the camera wake up to idle state again.
Worth trying

Reply:
It doesn't work, even if we sleep for 2 minutes and wait for the device to wake up.
Looks like this device is not supporting the manifest features.
I tried several waits/sleeps, from 5 seconds till 2 minutes, and non of them worked.